oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -0,0 +1,123 @@
+from oneview_redfish_toolkit.api.computer_system import ComputerSystem
+from oneview_redfish_toolkit import util
+
+
+computer_system = Blueprint("computer_system", __name__)
+
+
+@computer_system.route("/redfish/v1/Systems/<uuid>", methods=["GET"])
+def get_computer_system(uuid):
+    """Get the Redfish Computer System for a given UUDI.
+
+        Return ComputerSystem redfish JSON for a given
+        server harder UUID.
+        Logs exception of any error and return abort(500)
+        Internal Server Error.
+
+        Returns:
+            JSON: Redfish json with ComputerSystem
+            When Server hardware is not found calls abort(404)
+
+
+        Exceptions:
+            Logs the exception and call abort(500)
+
+    """
+    try:
+        # Recover OV connection
+        ov_client = util.get_oneview_client()
+
+        # Gets serverhardware for given UUID
+        sh = ov_client.server_hardware.get(uuid)
+
+        # Gets de server hardware type of the given server hardware
+        sht = ov_client.server_hardware_types.get(sh['serverHardwareTypeUri'])
+
+        # Build Computer System object and validates it
+        cs = ComputerSystem(sh, sht)
+
+        # Build redfish json
+        json_str = cs.serialize()
+
+        # Build response and returns
+        return Response(
+            response=json_str,
+            status=status.HTTP_200_OK,
+            mimetype="application/json")
+    except HPOneViewException as e:
+        if e.error_code == "RESOURCE_NOT_FOUND":
+            if e.msg.find("server-hardware-types") >= 0:
+                logging.warning(
+                    'ServerHardwareTypes ID {} not found'.
+                    format(sh['serverHardwareTypeUri']))
+            else:
+                logging.warning(
+                    'ServerHardware UUID {} not found'.
+                    format(uuid))
+            abort(status.HTTP_404_NOT_FOUND)
+        elif e.msg.find("server-hardware-types") >= 0:
+            logging.error(
+                'OneView Exception while looking for server hardware type'
+                ' {}'.format(e)
+            )
+            abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+        elif e.msg.find("server-hardware") >= 0:
+            logging.error(
+                'OneView Exception while looking for '
+                'server hardware: {}'.format(e)
+            )
+            abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+        else:
+            logging.error('Unexpected OneView Exception: {}'.format(e))
+            abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+    except Exception as e:
+        # In case of error print exception and abort
+        logging.error('Unexepected error: '.format(e))
+        return abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@computer_system.errorhandler(status.HTTP_404_NOT_FOUND)
+def not_found(error):
+    """Creates a Not Found Error response"""
+    return Response(
+        response={'error': 'URL not found'},
```

Comment:
I think that the response should be like this: '{"error": "URL not found"}' because it represent a JSON string.
When you add the response.data test in ComputerSystemTest class you'll realize this mistake.